ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -589,13 +607,15 @@ PrjFS_Result PrjFS_DeleteFile(
         PrjFSFileXAttrData xattrData = {};
         if (!TryGetXAttr(fullPath, PrjFSFileXAttrName, sizeof(PrjFSFileXAttrData), &xattrData))
         {
+            LogWarning("PrjFS_DeleteFile: failing because we were unable to get PrjFSFileXAttrName fullPath=%s", fullPath);
             *failureCause = PrjFS_UpdateFailureCause_FullFile;
             return PrjFS_Result_EVirtualizationInvalidOperation;
         }
     }
     
     if (0 != remove(fullPath))
     {
+        LogWarning("PrjFS_DeleteFile: remove failed fullPath=%s errno=%d strerror=%s", fullPath, errno, strerror(errno));
```

Comment:
Similar to comment above, we should not log this message in the cases where the function would return `PrjFS_Result_Success`.

I think we could get by only logging this in the `PrjFS_Result_EIOError` case